src/elements/content-sidebar/additional-tabs/AdditionalTabFtuxTooltip.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @flow
+ * @file Sidebar Additional Tab FTUX tooltip
+ * @author Box
+ */
+
+import * as React from 'react';
+
+import TargetedClickThroughGuideTooltip from '../../../features/targeting/TargetedClickThroughGuideTooltip';
+
```

Comment:
Is the whitespace between each import needed? Most of our other files group imports together.

Reply:
I was trying to order the imports similarly to how we do it in EUA, but you're right - the other files don't do that. I'll get rid of the whitespace!